### ISSUE TITLE
Deprecate base64 Textures

### ISSUE THREAD

**Issue by menduz** [decentraland/explorer-technical-backlog#24]
Tracked in https://github.com/decentraland/sdk/issues/64

**Comment by leanmendoza** [comment]
What about Poap and EventThumbnail textures? https://github.com/decentraland/sdk/issues/64
Should all this new components have the `Texture` suffix?

**Comment by menduz** [comment]
Yes, let's move the conversation to https://github.com/decentraland/sdk/issues/64
